pkgs/by-name/ax/axmldec/package.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, cmake, doxygen, boost, zlib }:
```

This is completely optional, but laying out the arguments with newlines make it easier for reviewers to look and comment on diffs later

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, cmake
, doxygen
, boost
, zlib
}:
```